auto_round/export/export_to_autoround/__init__.py

```diff
@@ -12,11 +12,16 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+from ...utils import logger
 import transformers
-from .autoround_quantizer import AutoHfQuantizer
-
-transformers.quantizers.auto.AutoHfQuantizer = AutoHfQuantizer
-transformers.quantizers.auto.AutoQuantizationConfig = AutoHfQuantizer
-transformers.modeling_utils.AutoHfQuantizer = AutoHfQuantizer
+version = [int(item) for item in transformers.__version__.split('.')[:2]]
+if version[0] == 4 and version[1] < 38:
+    logger.warning("If you need to use the 'gpu' format to export layers like 'lm_head'," \
```

Comment:
not a good design, when using transformers <4.38, it always report this warning even users don't use lm-head quantization